package-lock.json

```diff
@@ -71,12 +71,12 @@
       }
     },
     "@babel/generator": {
```

Comment:
looks like even more changes than last time ;)

try `git commit master -- ./package-lock.json` and commit the updated file.